src/jdcal/__init__.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
This file is a package and has no external side effects, it should not be executable as is.

```suggestion
```